### PR TITLE
London Class 8 - Yigit Demirer - HTML/CSS - Week 2

### PR DESCRIPTION
**Pull request by YDemirer**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Yigit Demirer
- Your City: London
- Your Slack Name: YDemirer

# Homework Details

- Module: HTML/CSS
- Week: 2

# Notes

- What did you find easy?
Using semantic HTML tags
Creating links, buttons, and adding images

- What did you find hard?
Nesting containers

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Comment by moneyinthesky** [comment]
Hey @YDemirer, the end result looks great! Just a couple of points for feedback:
- I like how you used some semantic tags like header, footer and main. Do you think you could use any other tags like this? Maybe section?
- In the future, it might be a good idea to reference these background images (like first-background.jpg) in the CSS, and then you can avoid using position: absolute. I think we had this discussion in yesterday's class.

**Comment by YDemirer** [comment]
> Hey @YDemirer, the end result looks great! Just a couple of points for feedback:
> 
> * I like how you used some semantic tags like header, footer and main. Do you think you could use any other tags like this? Maybe section?
> * In the future, it might be a good idea to reference these background images (like first-background.jpg) in the CSS, and then you can avoid using position: absolute. I think we had this discussion in yesterday's class.

I am so sorry! I've only just seen your feedback. I will work on the points you mentioned. Thanks a lot.

File: pages/store.html [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    <a href="#help">Help</a>
+    <a href="#login">Login</a>
+</nav>
+    </header>
+
+<main> 
+    <div class="outer_container_2">
+        <div class="order_form">
+          <h3 class="orderkarma">Order your Karma wifi <br>device today!</h3>
+          <form>
```
Comment (moneyinthesky) [inline pages/store.html:30]: Nicely laid out form, well done.

File: pages/store.html [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    <a class="active" href="#meetkarma">Meet Karma</a>
+    <a href="#howitworks">How it Works</a>
+    <a class="active2" href="#store">Store</a>
+    <a href="#blog">Blog</a>
+    <a href="#help">Help</a>
+    <a href="#login">Login</a>
+</nav>
+    </header>
+
+<main>
```
Comment (moneyinthesky) [inline pages/store.html:26]: Good use of semantic HTML tags throughout the page

File: pages/store.html [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+<main> 
+    <div class="outer_container_2">
+        <div class="order_form">
+          <h3 class="orderkarma">Order your Karma wifi <br>device today!</h3>
+          <form> 
+            
+                <div class="fullname">
+                  <p>
+                      <label for="first_name">First name<span class="req">*</span></label><br>
+                      <input type="text" class="fname" name="first_name"/>
```
Comment (moneyinthesky) [inline pages/store.html:35]: Maybe you can think about validation attributes that can be added here?

**Comment by moneyinthesky** [comment]
Hey @YDemirer, overall this is looking quite good, well done! If you get a chance, definitely consider adding some validation on the input fields. You can read more about this here - https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#using_built-in_form_validation. Please let me know if you have questions about this!

File: css/style.css [hunk 4]
```diff
@@ -17,3 +17,388 @@ body {
...
+}
+
+header {
+    height: 6em;
+    padding-left: 6em;
+    background-color: rgba(196, 196, 196, 0.1);
+}
+
+nav {
+    float: right;
```
Comment (Monika39-maker) [inline css/style.css:34]: Lucy said, 'float' property is not very much in trend anymore. so, may be not use it

**Review by Monika39-maker — CHANGES_REQUESTED** [review]
I added a comment. please check. and confirm it from others  fo me as well 😆

**Pull request closed without merging**